dotcom-rendering/src/layouts/GalleryLayout.tsx

```diff
@@ -1,13 +1,18 @@
 import { css } from '@emotion/react';
+import { from } from '@guardian/source/foundations';
+import { ArticleHeadline } from '../components/ArticleHeadline';
+import { MainMediaGallery } from '../components/MainMediaGallery';
 import { Masthead } from '../components/Masthead/Masthead';
 import { grid } from '../grid';
+import type { ArticleFormat } from '../lib/articleFormat';
 import type { NavType } from '../model/extract-nav';
 import type { Gallery } from '../types/article';
 import type { RenderingTarget } from '../types/renderingTarget';
 
 interface Props {
 	gallery: Gallery;
 	renderingTarget: RenderingTarget;
+	format: ArticleFormat;
```

Comment:
We probably don't need this `format` prop given `gallery` contains format? It could also cause problems if the two representations were to end up out of sync?